apps/chat/src/utils/app/data/storages/api/conversation-api-storage.ts

```diff
@@ -66,7 +66,12 @@ export const getOrUploadConversation = (
 
   if (conversation?.status !== UploadStatus.LOADED) {
     return forkJoin({
-      conversation: ConversationService.getConversation(conversation),
+      conversation: ConversationService.getConversation(conversation).pipe(
+        catchError((err) => {
```

Comment:
maybe it's better to pass this error on top level

Reply:
this method is used in epics directly and I handle `null`-case there with error notification. Here I just log error as it is